### ISSUE TITLE
The labels in some output formats for columns are incorrect.

### ISSUE THREAD

**Issue by ujtwr** [vincentarelbundock/modelsummary#761]
# Case1: kableExtra

There are extra vertical bars included in the column names.

```{r}
library(tidyverse)

d <-
  tibble(
    x = rnorm(100),
    y = rnorm(100)
  )

res <- lm(y ~ x, data = d)

modelsummary::modelsummary(
  res,
  output = "kableExtra",
  estimate = c(ABC = "estimate"),
  shape = term ~ model + statistic
)
```
![image](https://github.com/vincentarelbundock/modelsummary/assets/42717412/e5956142-2c2a-4bcb-8ccd-d949ba5d9e8a)

# Case2: flextable
There are extra slash included in the column names.

```{r}
library(tidyverse)

d <-
  tibble(
    x = rnorm(100),
    y = rnorm(100)
  )

res <- lm(y ~ x, data = d)

modelsummary::modelsummary(
  res,
  output = "flextable",
  estimate = c(ABC = "estimate"),
  shape = term ~ model + statistic
)
```
![image](https://github.com/vincentarelbundock/modelsummary/assets/42717412/1e193fee-3b00-4b6c-9b2c-3c79374a5982)

# Case3: gt
good!!

```{r}
library(tidyverse)

d <-
  tibble(
    x = rnorm(100),
    y = rnorm(100)
  )

res <- lm(y ~ x, data = d)

modelsummary::modelsummary(
  res,
  output = "gt",
  estimate = c(ABC = "estimate"),
  shape = term ~ model + statistic
)
```
![image](https://github.com/vincentarelbundock/modelsummary/assets/42717412/52bd57ce-4120-41af-ab68-251055641031)

**Comment by ujtwr** [comment]
Does this relate to anything?

https://github.com/haozhu233/kableExtra/issues/821

**Comment by vincentarelbundock** [comment]
Thanks for the report. This should be fixed in version 2.0.0.9 from Github.
